File: src/targets/php/guzzle/fixtures/jsonObj-multiline.php
<?php

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'http://mockbin.com/har', [
  'body' => '{
  "foo": "bar"
}',
  'headers' => [
    'content-type' => 'application/json',
  ],
]);

echo $response->getBody();

File: src/targets/csharp/restsharp/fixtures/jsonObj-multiline.cs
var client = new RestClient("http://mockbin.com/har");
var request = new RestRequest("", Method.Post);
request.AddHeader("content-type", "application/json");
request.AddParameter("application/json", "{\n  \"foo\": \"bar\"\n}", ParameterType.RequestBody);
var response = client.Execute(request);